include/net/coap.h

```diff
@@ -669,12 +670,14 @@ void coap_reply_init(struct coap_reply *reply,
  * confirmation message, initialized with data from @a request
  * @param request Message waiting for confirmation
  * @param addr Address to send the retransmission
+ * @param now Current time (in milliseconds)
```

Comment:
Why do we need to supply this in this API?

Reply:
I can just make CoAP call the kernel function if that's preferred.

Reply:
Done

Reply:
Hmm, did you send old version. The now parameter is still there.

Reply:
My bad, I've updated a wrong commit so it's added and then removed. Will fix.